Doc/library/asyncio-task.rst

```diff
@@ -447,8 +448,15 @@ Running Tasks Concurrently
    Tasks/Futures to be cancelled.
 
    .. note::
-      A more modern way to create and run tasks concurrently and
-      wait for their completion is :class:`asyncio.TaskGroup`.
+      A new alternative to create and run tasks concurrently and
```

Comment:
I don't think there should be this much information in a note.

Reply:
Thanks for your comment. It's difficult to turn into something actionable. Can you make a precise, constructive proposal?

Reply:
Honestly this belongs more to a FAQ entry than reference docs. But this note was already there, I suggest to only explain the case why gather should be used and keep it short something like "Use asyncio.gather if you want to concurrently run tasks and cancellation of one does not affects others otherwise it is recommend to use TaskGroups". Feel free to word it better!